resources/views/beatmappacks/show.blade.php

```diff
@@ -0,0 +1,47 @@
+{{--
+    Copyright 2015-2017 ppy Pty. Ltd.
+
+    This file is part of osu!web. osu!web is distributed with the hope of
+    attracting more community contributions to the core ecosystem of osu!.
+
+    osu!web is free software: you can redistribute it and/or modify
+    it under the terms of the Affero GNU General Public License version 3
+    as published by the Free Software Foundation.
+
+    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+    See the GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+--}}
+
+
+@if(Auth::check())
+<ul class="beatmap-pack__downloads">
+    @foreach ($pack->downloadUrls() as $download)
+        <li class="beatmap-pack__download">
+            <a href="{{ $download['url'] }}"
+                class="beatmap-pack__link">Download
```

Comment:
the closing tag should be immediate after the text (which should be translated) otherwise fun thing happens ![](https://s.myconan.net/2017-07/firefox_2017-07-19_16-54-46.png).

Also the design doesn't seem to match the psd file? Download link is larger than host and bolded.